warehouse/templates/packaging/detail.html

```diff
@@ -244,7 +244,8 @@ <h2 class="no-top-padding">Reverse Dependencies (TODO)</h2>
             <div class="release{{ ' release--latest' if loop.first else ' release--oldest' if loop.last }}{{ ' release--current' if is_current }}">
               <div class="release__meta">
                 {% if is_current %}
-                <p class="release__version">{{ hrelease.version }}</p>
+                <p class="release__version"><a href="{{ request.route_path('packaging.release', name=release.project.name,
+                version=hrelease.version) }}">{{ hrelease.version }}</a></p>
```

Comment:
I'm sorry this will be a super silly question but where are you seeing the `AssertionError`? It didn't show when I ran `make tests` so I'm assuming I'm missing something. 
I will go back and try again. Thanks! :)